### DISCUSSION TITLE
Generated documentation contains "x-tokenname" in lowercase instead of "x-tokenName"

### DISCUSSION THREAD

**Discussion by akovalchuk** [swaggo/swag#1661] [Q&A]
Hello,

Generated documentation contains "x-tokenname" in lowercase instead of "x-tokenName" and it prevents Swagger UI to work correctly.

I tried to use the following annotations:
- //	@x-tokenname	id_token
- //	@x-tokenName	id_token

>swag init 
Creates "docs.go", "swagger.json", "swagger.yaml" files with "x-tokenname": "id_token" in lowercase. 

I have checked the source code of the "swagger-ui-bundle.js" and it expects for the "x-tokenName" (camelcase) 

After updating manually all three files and replacing  "x-tokenname" -> "x-tokenName" may start to send & fetch correct attribute.

Does anyone know why it happens and how to solve that?

Thank you.

P.S.: OS - Windows 11 Pro, swag.exe version v1.8.12

**Comment by willforeverin** [comment]
I hit the same issue. I'm updating `docs.SwaggerInfo.SwaggerTemplate` by replacing `x-tokenname` to `x-tokenName` to work-around it.
I think the right solution is to change the swagger-ui-bundle.js to use `x-tokenname` instead. 
is it repo still active? feels confused issue like this reported for almost 1 year and not get answered/fixed.
